crates/iota-sdk-types/src/crypto/signature.rs

```diff
@@ -212,16 +212,23 @@ impl SimpleSignature {
 /// zklogin-flag     = %x05
 /// passkey-flag     = %x06
 /// ```
-#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord, derive_more::Display)]
```

Comment:
derive_more::Display is a bit annoying with the manual lowercasing. One alternative would be to use `strum` and add:

```
#[derive(strum::Display)]
#[strum(serialize_all = "lowercase")]
```

to the enum to make it less noisy and typo-resistent.

Reply:
I think I will address this in a new PR, because there are a few places we could do this